sdk/tables/azure-data-tables/README.md

```diff
@@ -25,17 +20,287 @@ best parts of Azure cloud.
+- Storing datasets that don't require complex joins, foreign keys, or stored procedures and can be de-normalized for fast access
+- Quickly querying data using a clustered index
+- Accessing data using the OData protocol
+
 ## Examples
-Use the client library for Tables to:
-- manages tables
-- edit and access table contents
+
+- [Authenticate a client](#authenticate-a-client)
+  - [Authenticate with a connection string](#authenticate-with-a-connection-string)
+  - [Authenticate with a Shared Key](#authenticate-with-a-shared-key)
+  - [Authenticate with a Shared Access Signature (SAS)](#authenticate-with-a-shared-access-signature-sas)
+- [Create, List, and Delete Azure tables](#create-list-and-delete-azure-tables)
+  - [Construct a `TableServiceClient`](#construct-a-tableserviceclient)
+  - [Create a table](#create-a-table)
+  - [List tables](#list-tables)
+  - [Delete a table](#delete-a-table)
+- [Create, List, and Delete table entities](##create-list-and-delete-table-entities)
+  - [Construct a `TableClient`](#construct-a-tableclient)
+  - [Create an entity](#create-an-entity)
+  - [List entities](#list-entities)
+  - [Delete an entity](#delete-an-entity)
+
+### Authenticate a client
+
+#### Authenticate with a connection string
+To use a connection string to authorize your client, call the builder's `connectionString` method with your connection string.
+
+```java
+TableServiceClient tableServiceClient = new TableServiceClientBuilder()
+    .connectionString("<your-connection-string>")
+    .buildClient();
+```
+
+#### Authenticate with a Shared Key
+To use a Shared Key to authorize your client, create an instance of `TablesSharedKeyCredential` with your account name and access key. Call the builder's `endpoint` method with your account URL and the `credential` method with the `TablesSharedKeyCredential` object you created.
+
+```java
+TablesSharedKeyCredential credential = new TablesSharedKeyCredential("<your-account-name>", "<account-access-key>");
+TableServiceClient tableServiceClient = new TableServiceClientBuilder()
+    .endpoint("<your-table-account-url>")
+    .credential(credential)
+    .buildClient();
+```
+
+#### Authenticate with a Shared Access Signature (SAS)
+To use a SAS to authorize your client, call the builder's `endpoint` method with your account URL and the `sasToken` method with your SAS.
+
+```java
+TableServiceClient tableServiceClient = new TableServiceClientBuilder()
+    .endpoint("<your-table-account-url>")
+    .sasToken("<sas-token-string>")
+    .buildClient();
+```
+
+### Create, List, and Delete Azure tables
+
+#### Construct a `TableServiceClient`
+Construct a `TableServiceClient` by creating an instance of `TableServiceClientBuilder` and then calling the builder's `buildClient` or `buildAsyncClient` methods.
+
+```java
+TableServiceClient tableServiceClient = new TableServiceClientBuilder()
+    .connectionString("<your-connection-string>") // e.g.
```

Comment:
looks like you intended to add connection string example after the `// e.g.` comment

Reply:
Oh, no these are examples of how to construct the client. Since I have a real example of each authentication method, when I got down to the examples for constructing a client I had to pick one, and I used `connectionString` as an example, hence the comment meant to indicate you would choose whichever authentication mechanism was appropriate.

Reply:
Maybe I can update the comment to something like `// or your preferred authentication method`

Reply:
Ah, ok - that makes sense. Yeah, I think the latter example comment would be more intuitive.

Reply:
Fixed